package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-access",
-  "version": "1.1.2",
+  "version": "2.0.0",
```

Comment:
Because you didn't break the older versions, I would be ok with it being a feature `1.2.0` - what do you think?

Reply:
I followed the same model we used on `react-localize`, but I'm ok with making it `1.2.0`.